package/MDAnalysis/topology/PDBQTParser.py

```diff
@@ -98,11 +98,16 @@ class PDBQTParser(TopologyReaderBase):
      - charges
 
     Guesses the following:
-     - elements
      - masses
 
     .. versionchanged:: 0.18.0
        Added parsing of Record types
+    .. versionchanged:: 2.7.0
+       Columns 67 - 70 in ATOM records, corresponding to the field *footnote*,
+       are now ignored. See Autodock's `reference`_.
+
+       .. _reference: https://autodock.scripps.edu/wp-content/uploads/sites/\
+       56/2021/10/AutoDock4.2.6_UserGuide.pdf
```

Comment:
It's ok to sometimes let long URLs hang over — that's cleaner than backslash-breaking
```suggestion
       .. _reference: 
          https://autodock.scripps.edu/wp-content/uploads/sites/56/2021/10/AutoDock4.2.6_UserGuide.pdf
```